great-idea/index.html

```diff
@@ -18,8 +18,89 @@
 
     <!-- Copy and paste your HTML from the first UI project here -->
       
+  <div class="container home">
+
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea!</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+  <header>
+    <nav class="header-item nav">
+      <a href="#">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+  </nav>
+
+  <img class="header-item logo" src="img/logo.png" alt="Great Idea! Company logo.">
+</header>
+
+  
+<div class="innovation-button-logo">
```

Comment:
I trred looking for innovation-button-logo on your css and I could not find it. Make sure you are selecting the right elements.